compute-checksum.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+wget https://github.com/Omniaevo/mqtt5-explorer/releases/download/v1.16.1/com.omniaevo.mqtt5-explorer.tar.gz
+sha256sum com.omniaevo.mqtt5-explorer.tar.gz
+
+rm com.omniaevo.mqtt5-explorer.tar.gz
```

Comment:
use x-checker to automate this, drop the script. https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#json-checker